tests/solana_tests/create_contract.rs

```diff
@@ -367,7 +367,7 @@ fn missing_contract() {
                 is_signer: false,
             },
         ])
-        .must_fail();
+        .may_fail();
```

Comment:
Missing check on result, This is exactly why `fn must_fail` should check the result itself